test/integration/scheduler/scheduler_test.go

```diff
@@ -705,13 +705,14 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			gomega.Expect(k8sClient.Create(ctx, wl1)).Should(gomega.Succeed())
 			framework.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, fooCq.Name, wl1)
 
-			ginkgo.By("Deleting foo flavors")
+			ginkgo.By("Deleting clusterQueue and foo flavors")
+			gomega.Expect(framework.DeleteClusterQueue(ctx, k8sClient, fooCq)).To(gomega.Succeed())
 			gomega.Expect(framework.DeleteResourceFlavor(ctx, k8sClient, fooFlavor)).To(gomega.Succeed())
 
 			ginkgo.By("Creating another workloads")
 			wl2 := testing.MakeWorkload("wl-2", ns.Name).Queue(fooQ.Name).Request(corev1.ResourceCPU, "1").Obj()
 			gomega.Expect(k8sClient.Create(ctx, wl2)).Should(gomega.Succeed())
-			framework.ExpectWorkloadsToBeFrozen(ctx, k8sClient, fooCq.Name, wl2)
+			gomega.Expect(wl2.Status.Conditions).Should(gomega.BeEmpty())
```

Comment:
I think this could be flaky. The workload controller would mark it as Pending because the cluster queue doesn't exist.

Reply:
Also, the name of this test doesn't correspond with what the test is doing. This is now testing that workloads are pending when the clusterqueue is deleted.

Reply:
Before https://github.com/kubernetes-sigs/kueue/pull/263, we can delete a resourceFlavor without the impact of a clusterQueue, but now we added the finalizer to resourceFlavor, we'll wait in termination until the referenced clusterQueue deleted first. The test clusterQueue Should be inactive when corresponding flavors deleted is no longer suitable any more. Submitted a cleanup PR here https://github.com/kubernetes-sigs/kueue/pull/294